README.en.md

```diff
@@ -0,0 +1,192 @@
+        <img src="image-doc/Screen-6.png" alt="Screen 6" width="45%"/>
+        <img src="image-doc/Screen-7.png" alt="Screen 7" width="45%"/>
+        <img src="image-doc/Screen-8.png" alt="Screen 8" width="45%"/>
+        <img src="image-doc/Screen-9.png" alt="Screen 9" width="45%"/>
+    </p>
+</div>
+
+**Screen 6 :** Click on the gray icon.
+
+**Screen 7 & 8 & 9 :** After modifying credentials, you'll see the node-RED files that have been modified in Local files. Just click on \+all to add and select the files. Then in **Screen 8** click on commit, a text box will appear. In it, write a description of what's been modified, a short explanation of what you've done (in our case, it might be update/credentials), then click on Commit. Now click on Commit History, which will open a tab showing your last commit and a button with two arrows, click on it, then on push.
+
+<img src="image-doc/Screen-10.png" alt="Screen 10" width="45%"/>
+
+**Screen 10:** Once step **screen 9** is complete, this pop-up asks you to enter your Github login details. Here, your Password is actually an access-token that you'll need to generate from Github. I refer you to the [documentation](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens\#creating-a-personal-access-token-classic) link, and once you've retrieved it and entered Password, click Retry.
+
+Once all these steps have been completed, you'll need to restart node-RED as you did in step **3.2 Restart Node-RED**.
+
+<div style="display: flex">
+    <p align="center">
+        <img src="image-doc/Screen-11.png" alt="Screen 11" width="45%"/>
+        <img src="image-doc/Screen-12.png" alt="Screen 12" width="45%"/>
+    </p>
+</div>
+
+**Screen 11:** Once the dashboard has been reopened, node-RED tells you that there are some missing dependencies. Just go to Manage project dependencies.
+
+**Screen 12:** Here you can see the two dependencies that node-RED offers. Just click on install, then Close, then refresh the web page.
+
+<div style="display: flex">
+    <p align="center">
+        <img src="image-doc/Screen-13.png" alt="Screen 13" width="45%"/>
+        <img src="image-doc/Screen-14.png" alt="Screen 14" width="45%"/>
+    </p>
+</div>
+
+**Screen 13:** You should have one dependency left: downloadfile. Then click on Manage project dependencies.
+
+**Screen 14:** Here you have the complete list of dependencies. You'll need to delete any that are no longer in use. In our case : 
+- node-red-contrib-dir2files,  
+- node-red-contrib-gpsd,  
+- node-red-contrib-python3-function,  
+- node-red-contrib-ui-multistate-switch,  
+- node-red-dashboard,  
+- node-red-node-pi-gpio,  
+- node-red-node-ui-list
+
+Then go to the Install tab, search for and install the downloadfile module *(@prescient-devices/node-red-contrib-downloadfile).* Once installed, you'll need to refresh the page and your online V2 installation with github project is complete.
```

Comment:
Try adding this dependency to the `project.json` file, and then maybe we can simplify this part of the readme.